### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by simtor**
Just trying to see if it works

### CODE REVIEW

**Comment by simtor** [comment]
Oops

File: src/Equality.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+console.log(1 === 1);
+// expected output: true
+
+console.log('hello' === 'hello');
+// expected output: true
+
+console.log('1' ===  1);
+// expected output: false
+
+console.log(0 === false);
```
Comment (TimboTambo) [inline src/Equality.js:27]: Might be nice for each of these examples to above console.log the calculation you're doing as a string (e.g. `console.log("0 == false"`) so when someone runs it they can see:
> 0 == "false"
> true

File: src/Equality.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  input : process.stdin, 
+  output : process.stdout
+});
+
+
+let num1 = Math.floor((Math.random() *10) + 1);
+let num2 = Math.floor((Math.random() *10) + 1);
+let answer = num1 + num2;
+
+rl.question(`What is ${num1} + ${num2}?\n`,
```
[the remark this answers is not in the document]
Comment (simtor) [reply src/Equality.js:42]: Line 39: It generates a random number between 0-10.
Line 40: Calculates the answer for two  numbers.
Line 42: Outputs the question to  be answered.

File: src/Equality.js (hunk 1 above)
Comment (simtor) [reply src/Equality.js:27]: That is a nice ideas I will do it

File: src/Equality.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (TimboTambo) [reply src/Equality.js:42]: But I mean - why is it in this examples file? It doesn't seem related to the rest of the code that's here and I don't think it teaches us anything about the difference between `==` and `===`.

[the remark this answers is not in the document]
Comment (simtor) [reply src/Equality.js:42]: I will delete it.

**Comment by simtor** [comment]
Made the changes

File: src/Equality.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+console.log("Equality (==)");
+
+console.log("1 == 1");
+console.log(1 == 1);
+// expected output: true
+
+console.log("hello == hello")
```
Comment (TimboTambo) [inline src/Equality.js:7]: I think the ' is missing from either side of the hello here - it should be `"'hello' == 'hello'"`. 
Same for some of the other examples below.

Comment (simtor) [reply src/Equality.js:7]: Yeah, I didn't  think about that

**Comment by simtor** [comment]
fixed the mispelling

**Pull request merged**